spec/backend/ssh/build_command_spec.rb

```diff
@@ -85,31 +85,5 @@
         expect(Specinfra.backend.build_command('test ! -f /etc/selinux/config || (getenforce | grep -i -- disabled && grep -i -- ^SELINUX=disabled$ /etc/selinux/config)')).to eq '/bin/sh -c test\ \!\ -f\ /etc/selinux/config\ \|\|\ \(getenforce\ \|\ grep\ -i\ --\ disabled\ \&\&\ grep\ -i\ --\ \^SELINUX\=disabled\$\ /etc/selinux/config\)'
       end
     end
-
-    context 'with sudo on alternative path' do
```

Comment:
Was this just a duplicate of 'with custom sudo path' ? https://github.com/serverspec/specinfra/pull/294/files#diff-c8a2c9482af2614c23632c267efa5b93L41